tools/mo/openvino/tools/mo/main.py

```diff
@@ -258,6 +258,10 @@ def raise_ie_not_found():
                 raise Error('Incorrect saved model tag was provided. Specify --saved_model_tags with no spaces in it')
             argv.saved_model_tags = argv.saved_model_tags.split(',')
 
+    if argv.mean_values and (argv.scale or argv.scale_values):
```

Comment:
Per my understanding expected behavior is that if user specified --mean_values, then --scale - it is correct order and warning is not needed. It should be shown only if user specified scale, then mean, e.g. `--scale <val> --mean_values <vals>`

Reply:
Check of arguments order makes logic much more complex. By default arguments parser don't guarantee saving arguments order. I don't think it is required, we have not "right" or "wrong" order, order of arguments just don't influence on order of applying and we warn about it if both arguments are in place. If somewhere logic of implementation will be changed, we just re-write message in warning.

Reply:
+ @slyalin Actually this is exactly what has been identified as requirement for 'MO' vs 'IE' preprocessing. Using IE API user is able to first scale input values, then apply means by simple writing `preprocess().scale(2).mean(3)`. And user may have wrong expectation passing `--scale 3 --mean_values 2` will do the same. But this expectation is incorrect and we should warn about this.
For regular case when user specifies mean, then scale, there is no warning needed, we shall have same behavior as before - `warning` means that user may do something wrong, but by specifying means, then scales they do everything correctly. @slyalin can you please comment if it is ok to always show warning in MO, if user specifies means and scales in one command line. For me it is very usual use case in computer vision, so unavoidable warning may be annoying